app/api/courses.py

```diff
@@ -14,7 +14,26 @@
 
 @router.post("/course")
 def post_course(inputs: NewCourseRequest, token: str = Depends(oauth2_scheme)):
-    """Create a new Course in Course Catalogue"""
+    """
+    Create a new Course in Course Catalogue\n
+    <b>course_name</b> is required.
```

Comment:
`industry_standards` and `course_name` are on the same line now. Please put \n at the end.